src/browser/Navigators.js

```diff
@@ -177,7 +178,26 @@ export const init =
       ( Card
       , deck
       , flags
-      )
+      );
+    const [deck3, $deck3] =
+      ( Runtime.env.url
+      ? Deck.open
+        ( Card
+        , deck2
+        , { output:
+            { uri: Runtime.env.url
+            , disposition: 'default'
+            , features: ''
+            , ref: null
+            , guestInstanceId: null
```

Comment:
`Navigators` is probably most complex  component as it delegates to polymorphic (a.k.a generic `Deck` component) and it may not be very clear what `Deck.open` should be passed. I'll try to explain:

So `Deck.open` has following type signature:
```js
type open = <action, model, flags>
  (api:Card<action, model, flags>, model:Model<model>, flags:flags):
  [Model<model>, Effects<Action<action, flags>>] 
```

Since `api` argument in this case is `Card` we can derive what the types of `action`, `model` and `flags` are from there. In this case [it's a `Navigator`](https://github.com/linclark/browserhtml/blob/a62789ecb28a310e51d447828bf22e1ff92d0fc1/src/browser/Navigators.js#L92-L98) and from it's [`init` function](https://github.com/linclark/browserhtml/blob/a62789ecb28a310e51d447828bf22e1ff92d0fc1/src/browser/Navigators/Navigator.js#L358) we can derive that [`Flags` type](https://github.com/linclark/browserhtml/blob/a62789ecb28a310e51d447828bf22e1ff92d0fc1/src/browser/Navigators/Navigator.js#L34-L41) is the type of the `flags` argument that needs to be passed to `Deck.open`. Inlining here for convenience:

```js
export type Flags =
  { output: Output.Flags
  , isPinned?: boolean
  , isInputEmbedded?: boolean
  , input?: Input.Flags
  , overlay?: Overlay.Flags
  , assistant?: Assistant.Flags
  }
```

As of `output` field it's type is defined in `WebView` [as `Flags` type](https://github.com/linclark/browserhtml/blob/a62789ecb28a310e51d447828bf22e1ff92d0fc1/src/browser/Navigators/Navigator/WebView.js#L49-L57) inlining it here for connivence:

```js
export type Flags =
  { uri: URI
  , disposition: Disposition
  , name: string
  , features: string
  , options?: {}
  , ref: any
  , guestInstanceId: ?string
  }
```

In this instance `name` field is missing in your change, which is what flow is attempting to tell in the following error:

```
src/browser/Navigators.js:101
  Member 1:
   29:   | { type: "Open", open: flags }
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ object type
  Error:
   35:   { output: Output.Flags
                   ^^^^^^^^^^^^ property `name`. Property not found in. See: src/browser/Navigators/Navigator.js:35
  188:             { uri: Runtime.env.url
                   ^ object literal. See: src/browser/Navigators.js:188
```

Rest of the error from flow is I believe are just a noise caused by this type mismatch.